cdc/capture.go

```diff
@@ -173,11 +173,13 @@ func (c *Capture) Run(ctx context.Context) (err error) {
 			}
 			failpoint.Inject("captureHandleTaskDelay", nil)
 			if err := c.handleTaskEvent(ctx, ev); err != nil {
-				select {
-				case <-c.session.Done():
-					log.Warn("handle task event failed because session is done", zap.Error(err))
+				lease, inErr := c.etcdClient.Client.TimeToLive(ctx, c.session.Lease())
```

Comment:
Could you add some caveats about `session.Done()`?

Reply:
addressed